.gitattributes

```
@@ -1,2 +1,3 @@
 dist/index.js    -diff -merge
 dist/index.js    linguist-generated=true
+.licenses/** -diff linguist-generated=true
```

a2f7e58a7ed03539b75521f4130bdf6b7ce9f942

```suggestion
.licenses/** -diff linguist-generated=true
```

---

Duplicate of #